client/sites/v2/site-overview/controller.tsx

```diff
@@ -0,0 +1,14 @@
+import { isEnabled } from '@automattic/calypso-config';
+import page from '@automattic/calypso-router';
+import type { Context as PageJSContext } from '@automattic/calypso-router';
+
+export function redirectToHostingDashboardBackportIfEnabled(
+	context: PageJSContext,
+	next: () => void
+) {
+	if ( isEnabled( 'dashboard/v2/backport/site-overview' ) ) {
+		return page.redirect( `/sites/v2/${ context.params.site }` );
```

Comment:
```suggestion
		return page.redirect( `/sites/${ context.params.site }` );
```

Reply:
oops...missing this due to rebasing 🤦‍♂️